src/utils/generateColors.ts

```diff
@@ -0,0 +1,27 @@
+import randomSeed from 'random-seed';
+
+export type SeededColorCombination = {
+    bg: string;
+    fg: string;
+}
+
+export function generateColors(seed: string): SeededColorCombination {
```

Comment:
I don't think this function should return a fg/bg pair. It greatly limits it's usefulness. I think having a function that can generate a random color from some seed is great, because it can be used in many cases, including to generate two colors (fg/bg).

Instead I think it should be renamed to `generateRandomColor()` and return just a single color.

Reply:
Let's figure this out together during a call!